libvast/src/system/sink_command.cpp

```diff
@@ -88,26 +88,19 @@ caf::message sink_command(const command::invocation& invocation,
   auto spawn_exporter
     = command::invocation{invocation.options, "spawn exporter", {*query}};
   VAST_DEBUG(&invocation, "spawns exporter with parameters:", spawn_exporter);
-  auto exp = spawn_at(self, node, spawn_exporter);
+  auto exp = spawn_at_node(self, node, spawn_exporter);
   if (!exp)
     return caf::make_message(std::move(exp.error()));
-  // Register the accountant at the Sink.
-  caf::error err;
-  self->request(node, infinite, get_atom::value)
-    .receive(
-      [&](const std::string& id, system::registry& reg) {
-        // Assign accountant to sink.
-        VAST_DEBUG(invocation.full_name, "assigns accountant from node", id,
-                   "to new sink");
-        auto er = reg.components[id].find("accountant");
-        if (er != reg.components[id].end()) {
-          auto accountant = er->second.actor;
-          self->send(snk, actor_cast<accountant_type>(accountant));
-        }
-      },
-      [&](error& e) { err = std::move(e); });
-  if (err) {
-    return caf::make_message(std::move(err));
+  {
```

Comment:
Can you add a comment on why this explicit scope is necessary (or, if it is not, remove it)?